### DISCUSSION TITLE
L10N に関するガイドラインを統一しませんか？

### DISCUSSION THREAD

**Discussion by kazushi3** [mozilla-japan/translation#721] [Ideas]
現在、こちらでは以下のガイドライン・表記ガイドが有効だと捉えています。初心者には差異が分かりにくく、相互に参照を求めている部分もあるため統一できればと OSC で話題になりました。統一して差し支えなければ、皆さんに意見をいただきながら草案を書こうと思います。何か統一できない理由があれば教えてください。

私が認識しているもの
- [Editorial Guideline](https://github.com/mozilla-japan/translation/wiki/Editorial-Guideline)
- [L10N Guideline](https://github.com/mozilla-japan/translation/wiki/L10N-Guideline)
- [Mozilla L10N Glossary](https://github.com/mozilla-japan/translation/wiki/Mozilla-L10N-Glossary)
- [日本語の文体](https://docs.google.com/spreadsheets/d/1y-hC-xMXawCgcYZwJDnvuSlAOTgMRLLyqXurpYkJbYE/htmlview#gid=0)
- [Firefox OS Glossary](https://github.com/mozilla-japan/translation/wiki/Firefox-OS-Glossary)
- [SUMO localization guide](https://github.com/mozilla-japan/translation/wiki/SUMO-localization-guide)

これだけ見ると用途別に見えますが、実際には各プロジェクトが複数のドキュメントの参照を求めます。
組み合わせの例:
[MDN](https://mozilla-japan.github.io/mdn-translation-guide/translation/6_edit-ja-file.html)
- Editorial Guideline
- L10N Guideline
- Mozilla L10N Glossary
- 日本語の文体
SUMO
MDN の 4 件 + SUMO Localization guide

この他にも、各「はじめかた」的な資料があります。

**Comment by gurezo** [comment]
@marsf @hATrayflood @hmatrjp @dynamis 
cc @potappo 

こちらのお話も本日の OSC Tokyo 2023 fall(2023.10.21) で、相談がありました。
ガイドラインは自分も気になっており、メンテナンスや管理も含めて議論が出来ればと考えております。
